apps/factory/src/server/api/trpc.ts

```diff
@@ -87,6 +89,8 @@ export const createTRPCContext = async (opts: CreateNextContextOptions) => {
     (res.getHeader("x-run-mode") as PromptRunModesType) ||
     PromptRunModesSchema.Enum.LOGGEDIN_ONLY;
 
+  const apiKey = req.headers.authorization?.split(" ")[1] as string;
+
```

Comment:
@pulkit21 this should be same as token -> const token = await getToken({ req }); 

Should not be needed